Traceback (most recent call last):
  File "/home/ubuntu/projects/wolf/data.provider/src/2.provider.py", line 26, in <module>
    kafka = KafkaClient("ec2-54-183-118-187.us-west-1.compute.amazonaws.com:9092")
TypeError: __init__() takes exactly 1 argument (2 given)
Traceback (most recent call last):
  File "/home/ubuntu/projects/wolf/data.provider/src/2.provider.py", line 29, in <module>
    kafka = KafkaClient(kafka_host + ":" + kafka_port)
TypeError: __init__() takes exactly 1 argument (2 given)
